plugins/module_utils/foreman_helper.py

```diff
@@ -1027,16 +1027,21 @@ def __init__(self, **kwargs):
 
     @property
     def entity_name_from_class(self):
-        """ Convert class name to entity name. The class name must follow folowing name convention:
-            * Starts with Foreman or Katello
-            * Ends with Module
-
-            This will concert ForemanMyEntityModule class name to my_entity entity name.
-            eg:
-            * ForemanArchitectureModule => architecture
-            * ForemanProvisioningTemplateModule => provisioning_template
-            * KatelloProductMudule => product
-            * ...
+        """
+        Convert class name to entity name.
```

Comment:
This is a property. Should it be "The entity name derived from the class name." ?

Reply:
Yes.
```suggestion
       The entity name derived from the class name.
```